studio/queues/local_queue.py

```diff
@@ -66,23 +66,24 @@ def is_active(self):
         return is_active
 
     def clean(self, timeout=0):
+        _ = timeout
         with _local_queue_lock:
             _, files = self._get_queue_status()
-            for f in files:
+            for one_file in files:
```

Comment:
does pylint not like one char variable names ?

Reply:
Yes, exactly.